test/e2e/middleware-general/app/middleware.js

```diff
@@ -74,9 +79,9 @@ export async function middleware(request) {
       console.log('missing ANOTHER_MIDDLEWARE_TEST')
     }
 
-    const { 'STRING-ENV-VAR': stringEnvVar } = process['env']
+    const { STRING_ENV_VAR: stringEnvVar } = process['env']
```

Comment:
@ijjk was this diff necessary?
the idea of this test was to test a `StringLiteral`, instead of checking for `Identifier` which is tested with the `ANOTHER_MIDDLEWARE_TEST` binding. 🙏

Reply:
It fails on deploy due to be an invalid env variable name in the CLI, we could skip testing that on deploy if that's preferable

Reply:
Ah no. It’s all good. I wish it wasn’t so tied to webpack and I could’ve unit tested it. Would made much more sense